presto-parquet/src/main/java/com/facebook/presto/parquet/writer/ParquetWriterOptions.java

```diff
@@ -22,6 +22,7 @@ public class ParquetWriterOptions
 {
     private static final DataSize DEFAULT_MAX_ROW_GROUP_SIZE = DataSize.valueOf("128MB");
     private static final DataSize DEFAULT_MAX_PAGE_SIZE = DataSize.valueOf("1MB");
+    private static final DataSize DEFAULT_MAX_DICTIONARY_PAGE_SIZE = DataSize.valueOf("1MB");
```

Comment:
In parquet-mr, we use DEFAULT_MAX_PAGE_SIZE for DEFAULT_MAX_DICTIONARY_PAGE_SIZE https://github.com/apache/parquet-mr/blob/9b5a962df3007009a227ef421600197531f970a5/parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java#L49. I don't know if we should be consistent for that. Are there cases where the two pages are set differently?

Reply:
You are right, here is no special reason to set different default value, so may be its better to be consist with parquet-mr, I have fixed that.